File: TestAppIntechRetrofit2/app/src/main/java/github/johnnysc/testappintechretrofit2/REST/SongList.java
package github.johnnysc.testappintechretrofit2.REST;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Hovhannes Asatryan on 12.08.16.
 */
public class SongList {
    @SerializedName("results")
    List<Song> songsList;

    public SongList(List<Song> songs){
        this.songsList = songs;
    }

    public List<Song> getSongsList(){
        return songsList;
    }
}

File: TestAppIntechRetrofit2/app/src/main/java/github/johnnysc/testappintechretrofit2/main/bean/SongList.kt
package github.johnnysc.testappintechretrofit2.main.di.bean

import com.google.gson.annotations.SerializedName

/**
 * @author Asatryan on 08.07.18
 */
data class SongList(@SerializedName("results") val songsList : List<Song>)